code/web/src/modules/auth/RoutePrivate.js

```diff
@@ -8,6 +8,7 @@ import { connect } from 'react-redux'
 import userRoutes from '../../setup/routes/user'
 
 // Component
+// if the user logs in succefully, they go to thier page, otherwise, they are directed to the login
 const RoutePrivate = (props) => (
   props.user.isAuthenticated
```

Comment:
I think the "?" is called a ternary operator in javascript. If you had to human speak these lines, how would you explain it? I understand conditionals, however, I'm having trouble with this code.
My attempt:
If the user is authenticated, 
then return props.role 
(it's else statement is ln 19)
  If you can props.role is equal to the user details role,
  then route to with the props component
  (it's else statement is ln 18)
     If the route for props component works, yay 🥳 
    (it's else statement is ln 17)